server/drivers/g15.c

```diff
@@ -202,61 +213,52 @@ int g15_convert_coords(int x, int y, int *px, int *py)
 MODULE_EXPORT void g15_chr (Driver *drvthis, int x, int y, char c)
 {
 	PrivateData *p = drvthis->private_data;
+	int px, py;
 
-	y--;
-	x--;
-	if ((x > p->width) || (y > p->height))
+	if (!g15_convert_coords(x, y, &px, &py))
 		return;
 
-	g15r_renderCharacterLarge(p->canvas, x, y, c, 0, 0);
+	/* Clear background */
+	g15r_pixelReverseFill(p->canvas, px, py, px + 7, py + 7, G15_PIXEL_FILL, G15_COLOR_WHITE);
```

Comment:
any reason you use literal 7s here instead of something involving G15_CELL_WIDTH/HEIGHT?